cloudify_cli/commands/install.py

```diff
@@ -0,0 +1,155 @@
+########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+"""
+Handles 'cfy install'
+"""
+
+import os
+import urlparse
+
+from cloudify_cli import utils
+from cloudify_cli.commands import blueprints
+from cloudify_cli.commands import deployments
+from cloudify_cli.commands import executions
+from cloudify_cli.constants import DEFAULT_BLUEPRINT_FILE_NAME
+from cloudify_cli.constants import DEFAULT_BLUEPRINT_PATH
+from cloudify_cli.constants import DEFAULT_INPUTS_PATH_FOR_INSTALL_COMMAND
+from cloudify_cli.constants import DEFAULT_INSTALL_WORKFLOW
+from cloudify_cli.exceptions import CloudifyCliError
+
+
+def install(blueprint_path, blueprint_id, validate_blueprint, archive_location,
+            blueprint_filename, deployment_id, inputs, workflow_id, parameters,
+            allow_custom_parameters, timeout, include_logs):
+
```

Comment:
wait.. so you have to supply the blueprint_id and deployment_id here? can't we deduce them and add a randomized prefix/suffix? I would think I'd be able to run `cfy install -p blueprint.yaml -i inputs.yaml` and that's it.

Am I missing something?